contents/startups-clerky.mdx

```diff
@@ -0,0 +1,313 @@
+          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">$55,000 in credit</h5>
+          <p class="m-0 text-[15px]">That's a lot of events, replays, API calls, and survey responses. It lasts a year!</p>
+        </li>
+        <li class="relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8">
+          <GatsbyImage image={getImage(props?.images[6])} alt="Startup merch" objectFit="contain" style={{ maxWidth: 200 }} />
+          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">Founder merch</h5>
+          <p class="m-0 text-[15px]">You can never have too many laptop stickers, hats, or free t-shirts, right? Right.</p>
+        </li>
+        <li class="relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8">
+          <GatsbyImage image={getImage(props?.images[15])} alt="Startup spotlight" objectFit="contain" style={{ maxWidth: 200 }} />
+          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">Partner benefits</h5>
+          <p class="m-0 text-[15px]">We're part of DigitalOcean's Hatch program. You'll get $25,000 of infra credit.</p>
+        </li>
+        <li class="relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8">
+          <GatsbyImage image={getImage(props?.images[16])} alt="Startup spotlight" objectFit="contain" style={{ maxWidth: 200 }} />
+          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">Startup spotlights</h5>
+          <p class="m-0 text-[15px]">Each month we feature a team in our startup newsletter. It could be you!</p>
+        </li>
+        <li class="relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8">
+          <GatsbyImage image={getImage(props?.images[9])} alt="Startup merch" objectFit="contain" style={{ maxWidth: 200 }} />
+          <h5 class="text-xl font-extrabold m-0 pb-1 pr-4">Referral perks</h5>
+          <p class="m-0 text-[15px]">Refer your accelerator to 2x your credits, or split $10k when you refer a friend.</p>
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="my-24 px-5">
+
+  <div class="max-w-screen-lg mx-auto font-bold">
+    
+    <blockquote class="text-2xl text-primary dark:text-primary-dark m-0 quote md:text-[32px] md:leading-tight">
+      <span class="leading-tight">
+        "Building is never just one-and-done. You always need to find ways to improve.
+        <span class="text-red"> PostHog is central to how we do that at Y Combinator</span>. It helps us try ideas, measure results and make better products.”
+      </span>
+      <footer class="flex items-center space-x-8 mt-9">
+        <GatsbyImage image={getImage(props?.images[4])} alt="Cat Li" objectFit="contain" style={{ maxWidth: 100 }} />
+        <span class="flex flex-col"><cite class="not-italic text-lg md:text-xl">Cat Li</cite><cite class="not-italic font-medium text-lg opacity-50 md:text-lg">Product &amp; Engineering Lead, <span class="inline-block">Y Combinator</span></cite></span>
```

Comment:
Should this be a different, Clerky related quote?

Reply:
Happy to take feedback from Clerky here when they review. Just need a quote we can use.